### ISSUE TITLE
Throws error when the view constains lazy

### ISSUE THREAD

**Issue by WolfDan** [eeue56/elm-static-html-lib#1]
This functionality has not yet been implemented in the library

Here and example:

```elm
[ ul [ class "menu" ] <|
      lazy2 Util.viewIf isLoading spinner
          :: navbarLink (page == Home) Route.Home [ text "Home" ] onUserClick
          :: viewSignIn page user onUserClick
]
```

Throw error: 

> ERROR:Expecting an object with a field named `node` at _.children[0].children[0].children[0].children[0].children[0].children[0].children[1].children[0].children[0] but instead got: {"type":"thunk","args":[false,{"type":"node","tag":"li","facts":{"className":"sk-three-bounce"},"children":[{"type":"node","tag":"div","facts":{"className":"sk-child sk-bounce1"},"children":[],"descendantsCount":0},{"type":"node","tag":"div","facts":{"className":"sk-child sk-bounce2"},"children":[],"descendantsCount":0},{"type":"node","tag":"div","facts":{"className":"sk-child sk-bounce3"},"children":[],"descendantsCount":0}],"descendantsCount":3}]}

**Comment by eeue56** [comment]
published as 0.0.9-alpha.1. Try it out and let me know if it works

**Comment by eeue56** [comment]
Closing as I bleieve this is resolved. Feel free to re-open with a minimal reproducable example if not.
